gunicorn_config.py

```diff
@@ -12,6 +12,7 @@
 bind = "0.0.0.0:{}".format(os.getenv("PORT"))
 statsd_host = "{}:8125".format(os.getenv("STATSD_HOST"))
 gunicorn.SERVER_SOFTWARE = 'None'
+preload_app = False
```

Comment:
Do we want this to be set to True like the PR suggests :trollface:

Reply:
no idea what you're talking about 😒